### PR TITLE
ci: pin the azcopy version used for the transaction reply job

### PR DESCRIPTION
**Pull request by bmwill**

### CODE REVIEW

File: docker/tools/Dockerfile [hunk 1]
```diff
@@ -33,7 +33,7 @@ RUN apt-get update && \
 RUN ln -s /usr/bin/python3 /usr/local/bin/python
 COPY docker/tools/boto.cfg /etc
 
-RUN cd /usr/local/bin && wget https://aka.ms/downloadazcopy-v10-linux -O- | tar --gzip --wildcards --extract '*/azcopy' --strip-components=1 --no-same-owner && chmod +x azcopy
+RUN cd /usr/local/bin && wget https://azcopyvnext.azureedge.net/release20210226/azcopy_linux_amd64_10.9.0.tar.gz -O- | tar --gzip --wildcards --extract '*/azcopy' --strip-components=1 --no-same-owner && chmod +x azcopy
```
Comment (sherry-x) [inline docker/tools/Dockerfile:36]: Confirmed this link is legit:
```
❯ curl -s -D- https://aka.ms/downloadazcopy-v10-linux
HTTP/1.1 301 Moved Permanently
Location: https://azcopyvnext.azureedge.net/release20210415/azcopy_linux_amd64_10.10.0.tar.gz
```

**Review by sherry-x — APPROVED** [review]

**Review by sausagee — APPROVED** [review]
Tested with canary.

**Pull request merged**